### ISSUE TITLE
Jupyter widget fails on Python 3.9

### ISSUE THREAD

**Issue by anpaz** [microsoft/quantum-viz.js#54]
**Describe the bug**

With Python 3.9, I get an error when trying to open the qviz widget on a Jupyter Notebook in the dev environment.

**To Reproduce**

I followed the steps in the CONTRIBUTING.md, namely install the dev environment using `poetry`:
```
pip install poetry nox nox-poetry
poetry install
```

Then, installed Jupyter Notebook with:
```
 pip install notebook
```

> During installation I see the versions of `traitlets` an `ipython` replaced, since the latest version of notebook for Python 3.9 **requires** a newer, incompatible, version:
```
 nbclient, ipykernel, argon2-cffi, nbconvert, notebook
  Attempting uninstall: traitlets
    Found existing installation: traitlets 4.3.3
    Uninstalling traitlets-4.3.3:
      Successfully uninstalled traitlets-4.3.3
  Attempting uninstall: ipython
    Found existing installation: ipython 7.16.1
    Uninstalling ipython-7.16.1:
      Successfully uninstalled ipython-7.16.1
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
quantum-viz 0.0.0 requires ipython<8.0.0, but you have ipython 8.3.0 which is incompatible.
```

After installing, I tried opening any of the notebooks in `quantum-viz/examples`, I get this error:
```
---------------------------------------------------------------------------
VarnameRetrievingError                    Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 Viewer(bell)

File C:\Repos\quantum-viz.js\quantum-viz\quantum_viz\widget.py:59, in Viewer.__init__(self, circuit, width, height)
     48 """
     49 Create Viewer instance.
     50 
   (...)
     56 :type height: int, optional
     57 """
     58 try:
---> 59     self.name = varname()
     60 except ImproperUseError:
     61     self.name = "_"
```

**Expected behavior**
The widget should work fine with Python 3.9

**Screenshots**
![image](https://user-images.githubusercontent.com/24311286/168941429-60ab23b6-b74b-4cbd-b21a-18631b7994c3.png)

**Additional context**
Add any other context about the problem here. e.g.
 - OS: windows
 - Python: 3.9
 - ipython = 8.3.0

**Comment by anpaz** [comment]
As a workaround, installing specific versions for notebook and ipython fixes the problem:

`pip install notebook==6.4.11 ipython==7.28.0`

**Comment by anpaz** [comment]
Fixed with #64
